src/cmd/dev.go

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The UDS Authors
+
+// Package cmd contains the CLI commands for UDS.
+package cmd
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	tea "github.com/charmbracelet/bubbletea"
+
+	"github.com/defenseunicorns/uds-cli/src/config"
+	"github.com/defenseunicorns/uds-cli/src/config/lang"
+	"github.com/defenseunicorns/uds-cli/src/pkg/bundle"
+	"github.com/defenseunicorns/uds-cli/src/pkg/bundle/tui/deploy"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	"github.com/defenseunicorns/zarf/src/pkg/utils/helpers"
+	"github.com/spf13/cobra"
+	"golang.org/x/term"
+)
+
+var devCmd = &cobra.Command{
+	Use:   "dev",
+	Short: lang.CmdDevShort,
+}
+
+var devDeployCmd = &cobra.Command{
+	Use:   "deploy",
+	Args:  cobra.MaximumNArgs(1),
+	Short: lang.CmdDevDeployShort,
+	PreRun: func(_ *cobra.Command, args []string) {
+		pathToBundleFile := ""
+		if len(args) > 0 {
+			if !helpers.IsDir(args[0]) {
+				message.Fatalf(nil, "(%q) is not a valid path to a directory", args[0])
+			}
+			pathToBundleFile = filepath.Join(args[0])
+		}
+		// Handle .yaml or .yml
+		bundleYml := strings.Replace(config.BundleYAML, ".yaml", ".yml", 1)
+		if _, err := os.Stat(filepath.Join(pathToBundleFile, config.BundleYAML)); err == nil {
+			bundleCfg.CreateOpts.BundleFile = config.BundleYAML
+		} else if _, err = os.Stat(filepath.Join(pathToBundleFile, bundleYml)); err == nil {
+			bundleCfg.CreateOpts.BundleFile = bundleYml
+		} else {
+			message.Fatalf(err, "Neither %s or %s found", config.BundleYAML, bundleYml)
```

Comment:
Something is fishy here, if I do a `uds dev deploy` in a directory with no `uds-bundle.yaml` then I don't get any error messages printed to the console, but if I look at the `uds logs` then I see them

Reply:
this one is interesting, since we only have the pretty tui setup for the deploy piece and not the create piece. May consider removing the "tea" for now

Reply:
removing tea for now, will add as part of follow on feature issue